synapse/models/economic.py

```diff
@@ -61,10 +67,29 @@ def getModelDefs(self):
                 ('econ:fin:tick', ('guid', {}), {
                     'doc': 'A sample of the price of a security at a single moment in time'}),
 
-                # econ:acct:bill
-                # econ:bank:us:aba:rtn ( ABA Routing Number )
-                # econ:bank:us:account = (econ:bank:us:aba:rtn, acct)
-                # econ:bank:swift:...
+                ('econ:bank:account:type:taxonomy', ('taxonomy', {}), {
+                    'doc': 'A bank account type taxonomy.'}),
+
+                ('econ:bank:account', ('guid', {}), {
+                    'doc': 'A bank account.'}),
+
+                ('econ:bank:balance', ('guid', {}), {
+                    'doc': 'A balance contained by a bank account at a point in time.'}),
+
+                ('econ:bank:statement', ('guid', {}), {
```

Comment:
should we consider econ:bank:acct:statement? The statement is linked to an acct

Reply:
It is commonly referred to as a "bank statement" and adding `account` would result in the double-nested name for the `:account` property which links it back to the `econ:bank:account` ( ie `econ:bank:account:statement:account` which is ugly ).